package/thingino-webui/files/var/www/x/j/reset-ntp.cgi

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if cp /rom/etc/ntp.conf /etc/ntp.conf; then
+if cp -f /etc/default/ntp.conf /tmp/ntp.conf; then
```

Comment:
firmware defaults is what comes in the firmware image. that information is available in /rom/. the file in /etc/default/ is changeable by user in runtime. when we want to reset the firmware, we go with the very original content.

Reply:
Got it.  Reset restores rom's NTP file to the working ntp file (tmp).  Should reset also erase user configuration in that default directory?